examples/meteringlab/c1/cluster.yaml

```diff
@@ -0,0 +1,183 @@
+# Namespace where the Scylla Cluster will be created
```

Comment:
it looks ok now, lets split it across multiple files/dirs each for each scylla cluster you're going to deploy - it needs small changes - separate ns, separate dns domains